internal/services/purview/purview_account_resource.go

```diff
@@ -94,6 +94,27 @@ func resourcePurviewAccount() *pluginsdk.Resource {
 				},
 			},
 
+			"managed_resources": {
+				Type:     pluginsdk.TypeList,
+				Computed: true,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"resource_group": {
+							Type:     pluginsdk.TypeString,
+							Computed: true,
+						},
+						"storage_account": {
+							Type:     pluginsdk.TypeString,
+							Computed: true,
+						},
+						"event_hub_namespace": {
+							Type:     pluginsdk.TypeString,
+							Computed: true,
+						},
```

Comment:
since these are all ID's, can we suffix these with `_id`:

```suggestion
						"resource_group_id": {
							Type:     pluginsdk.TypeString,
							Computed: true,
						},
						"storage_account_id": {
							Type:     pluginsdk.TypeString,
							Computed: true,
						},
						"event_hub_namespace_id": {
							Type:     pluginsdk.TypeString,
							Computed: true,
						},
```